### ISSUE TITLE
ASCII art formatting

### ISSUE THREAD

**Issue by Camology** [slaent/issues#94]
Disable the auto text formatting with a special tag that allows one to post ascii art. This can be exampled if you post the following string into a comment: 

██╗     ██╗ ███████╗ ██████╗   ███████╗
██║     ██║ ██╔════╝ ██╔══██╗ ██╔════╝
███████║ █████╗      ██████╔╝ █████╗
██╔══██║ ██╔══╝      ██╔══██╗ ██╔══╝
██║     ██║ ███████╗ ██║     ██║ ███████╗
╚═╝     ╚═╝ ╚══════╝ ╚═╝     ╚═╝ ╚══════╝ 

The letters will get all out of alignment because of (I assume) some text formatting you have behind the scenes. Allow the removal with a tag like [ascii] [/ascii] or whatever that makes the text plain jane.

**Comment by Camology** [comment]
And the text is all messed up on here too dang it

**Comment by todesignandconquer** [comment]
Haha! Can you give a concrete example of what it _should_ look like?

**Comment by Camology** [comment]
Yeah man, check out this link should show you how it should work out :  

http://patorjk.com/software/taag/#p=display&f=ANSI%20Shadow&t=H%20E%20R%20E

**Comment by todesignandconquer** [comment]
Ah, sounds good. Should be doable.

Does it work when using `[code]asci[/code]` by any chance?

**Comment by Camology** [comment]
Let me try hold on

**Comment by Camology** [comment]
Hey, yeah it kind of does! 
https://i.imgur.com/DMSehFT.png

Makes it white and red which is glaring on the dark theme, but the formatting does work. Awesome

**Comment by todesignandconquer** [comment]
Oh cool. Okay we can just piggy back off `code` tag then and create an `ascii` tag and make it non-red.

**Comment by Camology** [comment]
Very nice, this might be worthy of a second issue but maybe make that background color play off the chosen theme. IE: white for white theme, dark grey for dark theme? Just a thought.
